nzedb/IRCClient.php

```diff
@@ -519,7 +521,7 @@ protected function _ping($host)
 		$pong = $this->_writeSocket('PING ' . $host);
 
 		// Check if there's a connection error.
-		if ($pong === false || ((time() - $this->_lastPing) > ($this->_socket_timeout / 2) && !preg_match('/^PONG/', $this->_buffer))) {
+		if ($pong === false || ((time() - $this->_lastPing) > ($this->_socket_timeout / 2) && strpos($this->_buffer, 'PONG') === 0)) {
```

Comment:
I'm not sure the logic here is correct. The original checks if it is NOT found at the start, whereas your replacement looks for it at position zero. Shouldn't it be strpos($this->_buffer, 'PONG') !== 0)